index.html

```diff
@@ -0,0 +1,281 @@
+                    </li>
+                    <li id="tag3">
+                        <p id="tjavascript">javascript</p>
+                </ul>
+            </div>
+            <div id="action">
+                <button id="normal_button">
+                    <p id="see_project">See Project</p>
+                </button>
+            </div>
+        </div>
+        <section id="last_page">
+            <div id="about_myself">
+                <h2>About Myself</h2>
+                <p>Hello I'm a software developer! I can help you build a product, feature or website.
+                    Look through some of my work and exoerience!
+                    If you like what you see and have a project you need coded, don't hesitate to contact me.
+                </p>
+                <a href="#">LET’S CONNECT</a>
+                <ul class="social-links">
+                    <li>
+                        <a href="#"><img src="images/twitter.svg" alt="Twitter"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/linkedinicon.svg" alt="LinkedIn"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector (1).svg" alt="Angel List"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/github.svg" alt="Github"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector.svg" alt="Meduim"></a>
+                    </li>
+                </ul>
+
+            </div>
+            <button id="normal_button1">
+                <p id="resume">Get my resume</p>
+            </button>
+            <ul id="skill_list">
+                <li id="languages">
+
+                    <div id="langbox">
+                        <p id="lang_text">Languages</p>
+                        <img id="down_arrow" src="images/downarrow.svg">
+                    </div>
+                    <ul>
+                        <li><a id="langicon" href="#"><img src="images/javascript.svg" alt="javascript"></a></li>
+                        <li><a id="langicon" href="#"><img src="images/html.svg" alt="html"></a></li>
+                        <li><a id="langicon" href="#"><img src="images/css.svg" alt="css"></a></li>
+                    </ul>
+                </li>
+                <li id="frameworks">
+                    <div id="framebox">
+                        <p id="frame_text">Frameworks</p>
+                        <img id="right_arrow1" src="images/rightarrow.svg">
+                    </div>
+                </li>
+                <li id="skills">
+                    <div id="skillsbox">
+                        <p id="skills_text">Skills</p>
+                        <img id="right_arrow2" src="images/rightarrow.svg">
+                    </div>
+                </li>
+            </ul>
+        </section>
+
+        <section id="contact_me">
```

Comment:
Great job @Thinus01 🤝

- [ ] [Optional] As stated in the project requirements, kindly align the items of the contact form with flexbox.